go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/tinkerbell/rufio
 
-go 1.18
```

Comment:
Are we using 1.20 specific libraries or features that are requiring this to be bumped?

Reply:
No